docs/ref/models/querysets.txt

```diff
@@ -1708,6 +1708,13 @@ to refer to the queryset's model.
 
         Restaurant.objects.select_for_update(of=('self', 'place_ptr'))
 
+On PostgreSQL only, you can pass ``no_key=True`` in order to acquire the weaker
+lock. This means that creating rows that reference (through a foreign key)
+locked rows due to ``select_for_update(no_key=True)`` won't have to wait until
+the lock is released. The PostgreSQL documentation has more details about
+`row-level lock modes
+<https://www.postgresql.org/docs/current/explicit-locking.html#LOCKING-ROWS>`_.
```

Comment:
I think at least `acquire the weaker lock` needs to be `acquire a weaker lock`. _What lock?_, but maybe: 

```suggestion
On PostgreSQL only, you can pass ``no_key=True`` in order to acquire a weaker
lock, that still allows the creating rows that merely reference locked rows
(through a foreign key, for example) whilst the lock is in place. The
PostgreSQL documentation has more details about `row-level lock modes
<https://www.postgresql.org/docs/current/explicit-locking.html#LOCKING-ROWS>`_.
```

It's a mouthful any which way...